### ISSUE TITLE
yaourt destination directory is incorrectly set

### ISSUE THREAD

**Issue by shawngraham** [JonathanReeve/dh-usb#10]
so I thought, I'll give R a spin.... 

`pacman -S R` result:

pacman target not found
pacman failed to init transaction 
unable to lock database could not lock database: File exists

**Comment by shawngraham** [comment]
ah lord, it's lowercase r...

**Comment by shawngraham** [comment]
speaking of R, `yaourt rstudio-desktop.bin` seems to do nothing, but `yaourt rstudio-desktop` will give me the option of getting it; then it asks if i want to edit the pkgbuild (i say no), then 'you do not have write premission to store packages in /home/packages'

**Comment by shawngraham** [comment]
ah, ok - i had to cd up to the home directory (i was confused by dh-usb/home; needed to be at home/dh-usb ; then I made a packages directory, and gave it permissions with `sudo chmod -R 777 packages`; then yaourt rstudio etc works!

**Comment by JonathanReeve** [comment]
Thanks for the help testing all these commands! This is super helpful. I'll have to change the instructions to reflect the lowercase R. Rstudio should be installable with `yaourt rstudio-desktop-bin` (note the hyphen). You can get it to be silent about asking you if you want to edit pkgbuilds and stuff with `yaourt -S --noconfirm rstudio-desktop-bin`. I'll change the instructions to reflect that. 

That's strange about `yaourt` complaining about `/home/dh-usb/packages`. I think my custom yaourt config might be causing that. I put in a custom yaourt config in order to install AUR packages during install, but then I forgot to switch it back to default once that install is complete. I'll open a new issue for this. Thanks!
